mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
+    .exec()
+    .then(result => {
+      if (result.n === 0) {
+        return res.status(404).json({ message: "Order not found" });
```

Comment:
The response message is hardcoded in English. Consider using a translation library or a separate file for storing response messages to make the application more scalable and easier to maintain.